docs/tabpy-tools.md

```diff
@@ -30,6 +31,26 @@ The URL and port are where the Tableau-Python-Server process has been started -
 more info can be found in the
 [server section](server-startup.md#Command-Line-Arguments) of the documentation.
 
+## Authentication
+
+When TabPy is configured with authentication feature on client code
+has to specify credential to use during model deployment with
```

Comment:
This should be

When TabPy is configured with the authentication feature on, client code
has to specify the credentials to use during model deployment with

Reply:
fixed